framework/widgets/Menu.php

```diff
@@ -210,9 +210,9 @@ protected function renderItems($items)
             }
             if (!empty($class)) {
                 if (empty($options['class'])) {
-                    $options['class'] = implode(' ', $class);
+                    $options['class'] = $class;
                 } else {
-                    $options['class'] .= ' ' . implode(' ', $class);
```

Comment:
`$options['class']` is an array so it's not concatenated to `$class` but is merged with it.